lib/v2/screens/wallet/components/transactions_list/transactions_list.dart

```diff
@@ -55,7 +55,7 @@ class _TransactionsListState extends State<TransactionsList> with AutomaticKeepA
                       final account = settingsStorage.accountName;
                       final model = state.transactions[index];
                       return TransactionInfoRow(
-                        key: Key(model.transactionId),
+                        key: Key(model.transactionId ?? "index_$index"),
```

Comment:
I don't think anything happens if the index is used, is it given an identifier?

Reply:
It's fine, it just needs a unique key